src/store/features/featuresSlice.ts

```diff
@@ -3,15 +3,18 @@ import { getWallet } from '../../utils/walletClient'
 
 export type FeaturesState = {
   discoveryEnabled: boolean
+  checklistEnabled: boolean
```

Comment:
did you test to make sure that when the feature flag is true, checklist gets enabled?

Reply:
Yeah I mocked it locally and it worked